ui/src/app/components/runs/filters/datetime-range-filter/datetime-range-filter.component.ts

```diff
@@ -59,11 +59,11 @@ export class DatetimeRangeFilterComponent implements ClrDatagridFilterInterface<
   }
 
   isActive(): boolean {
-    return !!this.value.from || !!this.value.to
+    return !!this.value.from || !!this.value.to;
```

Comment:
If your IDE did that ok. Otherwise, I wouldn't make changes to files unrelated to the issue, because it makes the review more time-consuming

Reply:
I will start setting up a typescript plugin (if there is one) for IntelliJ instead of the VS Code.

Reply:
I've experimented with prettier, tslint and eslint here: 
https://github.com/AbsaOSS/hyperdrive-trigger/commit/d575ed6e308b4e381a78c7e174a53f6392908e99 and 
https://github.com/AbsaOSS/hyperdrive-trigger/commit/39396b59e77fc7cbc7c1e82584a842cbed8f96cc in one of the prototype branches for the new ui. We should definitely tackle this problem. Ideally, it would work both in IntelliJ and VSCode, if you prefer VSCode